src/dataflow/components/dataflow-program.tsx

```diff
@@ -189,12 +209,23 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     })();
   }
 
+  private runProgram = () => {
+    console.log(this.state.programRunTime);
+    this.setState({isProgramRunning: true});
+  }
+  private stopProgram = () => {
+    this.setState({isProgramRunning: false});
+  }
+  private setProgramRunTime = (time: number) => {
+    this.setState({programRunTime: time});
+  }
```

Comment:
Obviously these functions will need to grow in complexity, but for now they at least allow us to control the topbar state.